src/GraphQLParser.Tests/Visitors/SDLSorterTests.SortsSchemaDefinition.approved.KitchenSink.graphql

```diff
@@ -0,0 +1,142 @@
+{
+  query
+  unnamed(falsey: false, truthy: true)
+}
+
+# Copyright (c) 2015, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+query queryName($foo: ComplexType, $site: Site = MOBILE) {
+  whoever123is: node(id: [123, 456]) {
+    id
+    ...  @skip(unless: $foo) {
```

Comment:
There's an extra space here also, between `...` and `@skip`.

Reply:
#352